.github/workflows/ci.yml

```diff
@@ -244,8 +244,8 @@ jobs:
           key: ${{ runner.os }}-node-${{ needs.install-deps.outputs.node-version }}-modules-${{ hashFiles('package-lock.json') }}
       - name: Code coverage
         run: |
-          npx nx affected --target=test --browsers=ChromeHeadlessNoSandbox --code-coverage --parallel=3 --progress=false --source-map --watch=false
-          npx nx affected --target=posttest --browsers=ChromeHeadlessNoSandbox --parallel=3 --watch=false
+          npx nx affected --target=test --parallel=3 -c ci
+          npx nx affected --target=posttest --parallel=3 -c ci
```

Comment:
This approach normalizes the call between Karma and Jest test runners, moving runner-specific parameters to the project configuration.